python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/dlt/resource.py

```diff
@@ -90,7 +90,8 @@ def extract_resource_metadata(
             job
             for load_package in load_info_dict.get("load_packages", [])
             for job in load_package.get("jobs", [])
-            if job.get("table_name") == resource.table_name
+            if resource.table_name
+            in [table.get("table_name") for table in load_package.get("tables", [])]
```

Comment:
I'm curious if using `table_name` will work if a destination like s3 (or local filesystem) is used, since these aren't considered tables.

Would it be possible to get the failure for the resource name instead?